src/TodoAdd.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import "./style.css";
+import className from 'classnames';
+function TodoAdd({ setTodos }) {
+    const [inputValue, setInputValue] = useState("");
+    const [isInvalid, setInvalid] = useState(false);
+
+    const handleInputChange = (value) => {
+        setInputValue(value);
+        setInvalid(false);
+    };
+    const handleAddTodo = () => {
+        const trimmedValue = inputValue.trim();
+        if (trimmedValue) {
+            setTodos((prevTodos) => [
+                ...prevTodos,
+                { id: Date.now(), title: inputValue, completed: false },
+            ]);
+            setInputValue(""); 
+            setInvalid(false);  
+        }
+        else {
+            setInvalid(true); 
+        }
+    };
+
+    const inputClass = className(`form-control mb-3`,{
+        'is-invalid':isInvalid
+    });
+
+    const buttonClass = className("btn border border-2 btn-sm cursor-pointer");
```

Comment:
I do not understand why need to use this package. we can directly write it, also add classes condition easily in react, so why using an extra package? @kaushaki-kumari